qsiprep/interfaces/dwi_merge.py

```diff
@@ -714,3 +715,77 @@ def create_provenance_dataframe(
     image_df = pd.concat(series_confounds, axis=0, ignore_index=True)
     image_df["original_file"] = bids_sources
     return image_df
+
+
+class _PhaseToRadInputSpec(BaseInterfaceInputSpec):
+    phase_file = File(exists=True, mandatory=True)
+
+
+class _PhaseToRadOutputSpec(TraitedSpec):
+    phase_file = File(exists=True)
+
+
+class PhaseToRad(SimpleInterface):
+    """Convert phase image from arbitrary units (au) to radians.
+
+    This method assumes that the phase image's minimum and maximum values correspond to
+    -pi and pi, respectively, and scales the image to be between 0 and 2*pi.
+
+    Notes
+    -----
+    The code is derived from
+    https://github.com/nipreps/sdcflows/blob/c6cd42944f4b6d638716ce020ffe51010e9eb58a/\
+    sdcflows/utils/phasemanip.py#L26.
+
+    License
+    -------
+    Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
```

Comment:
Since it only applies to a specific function, I _think_ it only needs to go in the docstring. I took a look at https://www.nipreps.org/community/licensing/#example-of-our-expectations and will push some more comprehensive changes to the docstrings in a second.